lib/DeonRestClient.ts

```diff
@@ -98,11 +99,11 @@ const possiblyBadRequestOrNotFound = async (r: Response) => {
   throwUnexpected(r, data);
 };
 
-const idString = (id: string | ContractIdValue): string => {
+const idString = (id: string | PseudoValue<Pseudo.ContractId>): string => {
```

Comment:
Should be the aliases type name.

Reply:
Yeah,  I added these aliases quite late because I was unsure if the fact that these things are Pseudo(syntactic) should be hidden, but I think it's fine to do